From 12370b1970e1b9f487d031533e00ee434755985e Mon Sep 17 00:00:00 2001
From: David Shea <dshea@redhat.com>
Date: Thu, 24 Apr 2014 17:37:05 -0400
Subject: [PATCH 02/19] Rename flags to io_flags.

This variable conflicts with the flags parameter from
upstream commit 73bdf4ab6cab300f710d514a9e9fbd0f82318d6b, which I would
like to include.  I would not like to include
b45a24e9c89a14baf63bffe0a9ff04c1c1bffb29, since I don't want to deal
with removing options whether they actually work or not.
---
 modprobe.c | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/modprobe.c b/modprobe.c
index 2d05b9d..27c9bb2 100644
--- a/modprobe.c
+++ b/modprobe.c
@@ -754,7 +754,7 @@ static void rmmod(struct list_head *list,
 		  int ignore_commands,
 		  int ignore_inuse,
 		  const char *cmdline_opts,
-		  int flags)
+		  int io_flags)
 {
 	const char *command;
 	unsigned int usecount = 0;
@@ -800,7 +800,7 @@ static void rmmod(struct list_head *list,
 	/* Now do things we depend. */
 	if (!list_empty(list))
 		rmmod(list, NULL, 0, warn, dry_run, commands,
-		      0, 1, "", flags);
+		      0, 1, "", io_flags);
 	return;
 
 nonexistent_module:
@@ -1287,7 +1287,7 @@ static int handle_module(const char *modname,
 			  int strip_vermagic,
 			  int strip_modversion,
 			  const char *cmdline_opts,
-			  int flags)
+			  int io_flags)
 {
 	if (list_empty(todo_list)) {
 		const char *command;
@@ -1308,7 +1308,7 @@ static int handle_module(const char *modname,
 
 	if (remove)
 		rmmod(todo_list, newname, first_time, error, dry_run,
-		      commands, ignore_commands, 0, cmdline_opts, flags);
+		      commands, ignore_commands, 0, cmdline_opts, io_flags);
 	else
 		insmod(todo_list, NOFAIL(strdup(options)), newname,
 		       first_time, error, dry_run, modoptions,
@@ -1370,7 +1370,7 @@ int main(int argc, char *argv[])
 	char *newname = NULL;
 	char *aliasfilename, *symfilename;
 	errfn_t error = fatal;
-	int flags = O_NONBLOCK|O_EXCL;
+	int io_flags = O_NONBLOCK|O_EXCL;
 	int failed = 0;
 
 	/* Prepend options from environment. */
@@ -1442,7 +1442,7 @@ int main(int argc, char *argv[])
 			use_blacklist = 1;
 			break;
 		case 'w':
-			flags &= ~O_NONBLOCK;
+			io_flags &= ~O_NONBLOCK;
 			break;
 		case 'd':
 			nofail_asprintf(&dirname, "%s/%s/%s", optarg,
@@ -1580,7 +1580,7 @@ int main(int argc, char *argv[])
 					      commands, ignore_commands,
 					      ignore_inuse, strip_vermagic,
 					      strip_modversion,
-					      cmdline_opts, flags))
+					      cmdline_opts, io_flags))
 					failed = 1;
 
 				aliases = aliases->next;
@@ -1596,7 +1596,7 @@ int main(int argc, char *argv[])
 				      modoptions, commands,
 				      ignore_commands, ignore_inuse,
 				      strip_vermagic, strip_modversion,
-				      cmdline_opts, flags))
+				      cmdline_opts, io_flags))
 				failed = 1;
 		}
 	}
-- 
1.9.0

